README.md

```diff
@@ -55,28 +55,55 @@ getUser();
 ```
 **[⬆ back to top](#table-of-contents)**
 
-### Use searchable names
+### Use searchable names (part 1)
+
 We will read more code than we will ever write. It's important that the code we do write is 
 readable and searchable. By *not* naming variables that end up being meaningful for 
 understanding our program, we hurt our readers.
 Make your names searchable.
 
 **Bad:**
+
 ```php
-// What the heck is 86400 for?
-addExpireAt(86400);
+// What the heck is 448 for?
+$result = $serializer->serialize($data, 448);
+```
+
+**Good:**
 
+```php
+$json = $serializer->serialize($data, JSON_UNESCAPED_SLASHES | JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE);
 ```
 
-**Good**:
+### Use searchable names (part 2)
+
+**Bad:**
+
 ```php
-// Declare them as capitalized `const` globals.
-interface DateGlobal {
-    const SECONDS_IN_A_DAY = 86400;
+// What the heck is 4 for?
+if ($user->access & 4) {
+    // ...
+}
+```
+
+**Good:**
+
+```php
+class User
+{
+    const ACCESS_NONE = 0; // 0000
+    const ACCESS_READ = 1 << 0; // 0001
+    const ACCESS_CREATE = 1 << 1; // 0010
```

Comment:
Could there be normal integers and just 2 values? I think everything else is extra an not neccessary to uderstand the concept.

I like the example though

Reply:
fixed it